### ISSUE TITLE
All browser app crashing

### ISSUE THREAD

**Issue by vaibhav9596** [PixelExperience/android-issues#158]
## Build date

```
PixelExperience_Plus_lavender-11.0-20210526-1452-OFFICIAL
```

## Expected Behavior

```
 browse properly without force stop
```

## Current Behavior

```
I tried chrome and brave browser but unable to browse
```

## Possible Solution

```
should release update fast because browser is important for daily use
```

## Steps to Reproduce

```
1.After install Latest May
![photo_2021-06-09_21-28-47](https://user-images.githubusercontent.com/76656922/121393179-bcbc3000-c96d-11eb-9b80-3700af3ed068.jpg)
![photo_2021-06-09_21-28-47](https://user-images.githubusercontent.com/76656922/121393247-cb0a4c00-c96d-11eb-9f38-ecdd691b74ea.jpg)

 A11 plus build
2.
3.
4.
```

/device lavender
/version eleven_plus

**Comment by faham1997** [comment]
Is fixed on  June Security patch . Wait for update!

**Comment by victorgsfigueiredo** [comment]
Thank you @faham1997! I am getting the same issue then @vaibhav9596 . Waiting anxiously for the update!

**Comment by faham1997** [comment]
Has been fixed on PixelExperience_Plus_lavender-11.0-20210618-2115-OFFICIAL.zip

**Comment by vaibhav9596** [comment]
thank you
